src/app/components/terms-and-conditions/terms-and-conditions.component.html

```diff
@@ -3,40 +3,84 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">Terms and conditions</h1>
-        <p class="govuk-body">By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a routerLink="/privacy-policy">privacy policy</a>.</p>
-        <p class="govuk-body">The use of the Service includes the Manage Organisation and Manage Cases applications (sometimes known as ‘MyHMCTS’) and Judicial Case Manager (JCM).</p>
-        <p class="govuk-body">The term ‘you’ includes yourself and all users of the Service in your organisation.  The terms ‘we’ and ‘us’ refer to Her Majesty’s Courts and Tribunals Service.</p>
-        <p class="govuk-body">You also agree that you have a legitimate need to use the Service.</p>
+        <p class="govuk-body">By using this service you're agreeing to these terms of use. This includes the <a routerLink="/privacy-policy">privacy policy</a>.</p>
```

Comment:
check which is latest T&C's and update accordingly

Reply:
Done @paddybasi